jboss-cert-ldap-login-module/src/main/java/org/esbtools/auth/jboss/CertLdapLoginModule.java

```diff
@@ -160,15 +152,13 @@ public void initializeRolesProvider() throws Exception {
             LOGGER.debug("Certificate principal:" + certPrincipal);
 
             //first try getting search name from uid in certificate principle (new certificates)
```

Comment:
You can probably remove this comment now that there is only one way after removing support for legacy certificates